user/db.js

```diff
@@ -6,7 +6,11 @@ db.exec(`CREATE TABLE IF NOT EXISTS users (
         id INTEGER PRIMARY KEY AUTOINCREMENT,
         name TEXT NOT NULL UNIQUE COLLATE NOCASE,
         password TEXT NOT NULL,
-        versionkey INTEGER NOT NULL DEFAULT 1
+        versionkey INTEGER NOT NULL DEFAULT 1,
+        email TEXT NOT NULL UNIQUE COLLATE NOCASE,
+        created_on DATETIME DEFAULT CURRENT_TIMESTAMP,
+        recovery_email TEXT NOT NULL COLLATE NOCASE,
+        phone TEXT NOT NULL COLLATE NOCASE
```

Comment:
yeah, query-paging is not going to work unless you mod user_result_sets too.. shoulda tested it.